docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -737,6 +737,49 @@ The output of the query has the following columns:
     - ``boolean``
     - Whether or not the operation appended data
 
+Functions
+---------------
+
+The connector exposes following functions:
+
+table_changes
+^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries. This feature allows Delta tables to track row-level changes between versions of a Delta table.
+When enabled on a Delta table, it records "change events" for all the data written into the table.
+
+.. code-block:: sql
+
+    SELECT * FROM TABLE(system.table_changes('test_schema', 'test_table', since_version)
```

Comment:
I tried 

```
SELECT * FROM TABLE(system.table_changes(table_schema_name => 'test_schema', table_name => '" + tableName + "', since_version => -1))
```

Having named parameters seems a bit more expressive in the documentation.

BTW, let's change `table_schema_name` to `schema_name`